ProcessLib/Output.h

```diff
@@ -58,6 +58,15 @@ class Output
             const double t,
             GlobalVector const& x);
 
+    //! Writes output for the given \c process.
+    //! To be used for debug output after an iteration of the nonlinear solver.
+    void doOutputIteration(
```

Comment:
I would specify this is for nonlinear iterations, i.e. `doOutputNonlinearIteration`